### ISSUE TITLE
A11y_ScreenReader: Screen Readers (NVDA/JAWS) focus is not going over the chat, present in the chat bot in scan mode using up/down arrow key.

### ISSUE THREAD

**Issue by elopezanaya** [microsoft/BotFramework-WebChat#4898]
### Is it an issue related to Adaptive Cards?

No.

### What is the PWD impact?

Users who are dependent on-screen readers will find difficulty while navigating over the chat present in the chat bot.

### What browsers and screen readers do this issue affect?

Windows: Chrome with NVDA

### Are there any code-based customization done to Web Chat?

Yes, however, this also repro when customizations are not in use.

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Transcript navigation

### How to reproduce the issue?

just launch a LCW widget , is expected that all the information displayed as part of the chat window is part of the information narrated by screen reader

### What do you expect?

Screen Readers (NVDA/JAWS) focus should go over the chat in scan mode using up/down arrow key.

### What actually happened?

Screen Readers (NVDA/JAWS) focus is not going over the chat in scan mode using up/down arrow key.

If focus goes inside the chat bot, screen reader should announce complete information of the chat including name and time of the chat.
For example: hi 6:42 PM - sent like that

### Do you have any screenshots or recordings to repro the issue?

<img width="560" alt="image" src="https://github.com/microsoft/BotFramework-WebChat/assets/981914/d3571e97-ccb2-40bf-80cd-cc31b14863eb">

### Did you find any DOM elements that might have caused the issue?

The span with the time stamps info and user name is not reachable and somehow is not visible for the screen reader

### MAS reference

https://aka.ms/MAS1.3.1

**Comment by elopezanaya** [comment]
hi all, is there any update on this issue?

**Comment by elopezanaya** [comment]
hello., is this incident being reviewed or discarded?

**Comment by elopezanaya** [comment]
![image](https://github.com/microsoft/BotFramework-WebChat/assets/981914/cba44681-10e4-476e-b2e3-a128ab84a47b)

since the definition of the div content is linked via aria-labeldescribedby, when the time stamp is attached, this one is not included in the description of the screen reader
